app/javascript/react/utils/setSensor.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  FixedBasicParameterTypes,
+  MobileBasicParameterTypes,
+  ParameterType,
+  SessionType,
+  SessionTypes,
+} from "../types/filters";
+import { Sensor } from "../types/sensors";
+
+const setSensor = (
+  selectedParameter: ParameterType,
+  sessionType: SessionType,
+  sensors: Sensor[]
+) => {
+  const getSensor = (parameter: string) => {
+    const allSensors = sensors.filter(
+      (item) => item.measurementType === parameter
+    );
+
+    const firstSensor = allSensors[0];
+
+    return firstSensor;
+  };
+
+  if (sessionType === SessionTypes.FIXED) {
+    switch (selectedParameter) {
+      case FixedBasicParameterTypes.PARTICULATE_MATTER:
+        return {
+          sensorName: "Government-PM2.5",
```

Comment:
We have `"Government-PM2.5"` hardcoded in a few places. I would extract this to const and reuse